### DISCUSSION TITLE
eslint is not working properly

### DISCUSSION THREAD

**Discussion by dxy2233** [LazyVim/LazyVim#771] [Q&A]
Hello, I am new to neovim.
The default configuration, and according to the documentation to increase the settings of eslint.
Now only lua files can be formatted normally.

![屏幕截图 2023-05-18 130715](https://github.com/LazyVim/LazyVim/assets/19802295/3224553a-10f9-48fa-a9da-5ef94a939534)

This project can be formatted normally in vscode.
In addition, emmet_ls cannot be used after installing it through mason. Does it require additional configuration?

**Comment by folke** [comment]
https://www.lazyvim.org/configuration/recipes#use-eslint-for-fix-on-save-and-prettier-for-formatting

https://www.youtube.com/watch?v=a_ZpTPaSn38
